tests/foreman/cli/test_computeresource.py

```diff
@@ -108,7 +108,7 @@ def test_positive_create_with_name(self):
 
         """
         ComputeResource.create({
-            'name': gen_string(str_type='alpha'),
+            'name': 'cr {0}'.format(gen_string(str_type='alpha')),
```

Comment:
That is to ensure that a Compute Resource can also have spaces when being created. In this way we get coverage for the new behavior for both create and update.